### ISSUE TITLE
make install error

### ISSUE THREAD

**Issue by gonwalf** [RubyLane/rl_json#1]
Compiling works fine, but "make install" returns the following error:

make: INSTALL_DATA_DIR@: command not found

**Comment by cyanogilvie** [comment]
Thanks for reporting - was caused by an old TEA tclconfig version.  I've pulled in the latest version and fixed some other issues with the build system and it should work ok now.

It will still be tricky to build on systems that don't ship yajl >= 2.1 though (like Ubuntu 14.04).  If you build yajl from source and install it in the /usr prefix it should be fine, but there seems to be a bug in yajl where the values exported in its yajl.pc file conflict with the paths coded into its public headers.  It's still possible to install it to a non-system location and build this package against it, but it's not pretty.

The only thing I like less than fighting with autoconf is fighting with cmake (which yajl uses), so I haven't chased that further, but if you do need to build against a non-system libyajl, then what worked for me was to edit the yajl headers and remove their yajl/ prefixes, and let configure know about the location of yalj.pc using the PKG_CONFIG_LIBDIR env var.
